persist/sqlite/keys.go

```diff
@@ -0,0 +1,48 @@
+package sqlite
+
+import (
+	"crypto/ed25519"
+	"database/sql"
+	"errors"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/walletd/keys"
+)
+
+// AddSigningKey adds a signing key to the store. If the key already exists, it
+// is not added again.
+func (s *Store) AddSigningKey(sk types.PrivateKey) error {
```

Comment:
I don't love this tbh. Storing an unencrypted secret on disk. I'd like to throw the idea of a secret in the room that is passed in via env var and then use `Argon2id` or `scrypt` to derive entropy from it which we use for encrypting the private key. 
Sure when you lose that secret people get access to the keys anyway but I feel like people are at least slightly more aware of keeping their env vars secure compared to their file permissions.